tools/errorid.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
the filename is pretty generic so it's not easy to understand exactly what it does. And there is no comment that explains it neither. can it be clarified..

Reply:
I added some documentation in the `readme.md` of the `tools` folder.

Reply:
- compare-errorids
- check-errorids
- check-errorid-coverage

Definitely something containing "errorid".

Reply:
I don't like "check-errorids" . but no strong opinion about those other.

imho I would add `testsuite-errorlist` into your names.
 * compare-errorids-testsuite-errorlist
 * check-errorid-coverage-testsuite-errorlist

the names are a bit longish but imho this is fine.